compiler/src/main/scala/org/scalaexercises/exercises/compiler/Compiler.scala

```diff
@@ -124,18 +123,20 @@ case class Compiler() {
     def enhanceDocError(path: List[String])(error: String) =
       s"""$error in ${path.mkString(".")}"""
 
-    def maybeMakeLibraryInfo(
-        library: Library
-    ) =
+    def maybeMakeLibraryInfo(library: Library) =
       for {
         symbol ← internal.instanceToClassSymbol(library)
         symbolPath = internal.symbolToPath(symbol)
-        comment ← (internal.resolveComment(symbolPath) >>= Comments.parseAndRender[Mode.Library])
-          .leftMap(enhanceDocError(symbolPath) _)
-        sections ← checkEmptySectionList(symbol, library) >>= {
+        comment ← (internal
```

Comment:
we don't want unicode symbols

Reply:
Just out of curiosity @kiroco12, was this caused by some IntelliJ plugin or what?

Reply:
No @calvellido, the code compiles with the unicode characters. But Adri prefer to use '=>' or '<-'

Reply:
But do you write them yourself with some key combination or what? 🤯

If you want the best of both worlds, you can us e afont with ligatures, like Fira Code:

https://github.com/tonsky/FiraCode

Reply:
That's something related to scalariform, I wouldn't care about that for now. Replace scalariform by scalafmt is probably a different task.